cmd/osquery-perf/agent.go

```diff
@@ -946,6 +957,54 @@ func (a *agent) runMacosMDMLoop() {
 				// Note: Declarative management could happen async while other MDM commands proceed. This is a potential enhancement.
 				a.doDeclarativeManagement(mdmCommandPayload)
 				mdmCommandPayload = nextMdmCommandPayload
+			case "InstalledApplicationList":
+				var installedVPPSoftware []fleet.Software
+				for _, adamID := range a.installedAdamIDs {
+					if sw, ok := adamIDsToSoftware[adamID]; ok && sw != nil {
+						if sw.Installed {
```

Comment:
Probably comment to explain what this is doing ("this will simulate an install that isn't done the first time we check, and then say it's installed next time")